src/applications/vaos/sass/vaos.scss

```diff
@@ -173,18 +182,35 @@
   display: flex;
   flex-wrap: wrap;
   margin: 0;
+  margin-top: 8px;
+
+  @include media($small-screen) {
+    margin-top: 0;
+  }
+}
+
+.vaos-calendar__option-cell {
```

Comment:
I might be able to do some of this with the flex grid, will look into that today

Reply:
This doesn't really work because 5 doesn't divide into 12, actually.